db/system_distributed_keyspace.cc

```diff
@@ -246,6 +287,12 @@ future<> system_distributed_keyspace::remove_view(sstring ks_name, sstring view_
  * use CL = ONE for writing the data, which will do the right thing -- saving the data in the only
  * possible replica. Until another node joins, reads will also use CL = ONE, retrieving the data
  * from the only existing replica.
+ *
+ * With system_distributed_v2 tables things are simpler since they are using the Everywhere
```

Comment:
Maybe name the keyspace somehow more meaningful? For example, system_distributed_everywhere?